ts/test/fixture/zomes/coordinator/src/lib.rs

```diff
@@ -42,92 +41,3 @@ fn signal_loopback(value: LoopBack) -> ExternResult<()> {
     emit_signal(&value)?;
     Ok(())
 }
-
-// #[hdk_extern]
-// fn create_two_party_countersigning_session(
```

Comment:
Oh interesting, did this get commented out because of the unstable features? I quite wanted a demo of this working in tryorama.

Reply:
Until we decide to remove the feature, could this code be kept?

Reply:
Yes, because of unstable features. Sure https://github.com/holochain/tryorama/pull/265/commits/69619edc3fe573481b255f3c19441bf73a721766